doc-content/enterprise-only/openshift/operator-modify-proc.adoc

```diff
@@ -0,0 +1,49 @@
+[id='operator-modify-proc']
+= Modifying an environment that is deployed using operators
+
+If an environment is deployed using operators, you can not modify it using typical OpenShift methods. For example, if you delete a pod, it is re-created automatically with the same parameters.
+
+To modify the environment, you must modify the YAML description of the environment. You can change common settings, such as passwords, add new {KIE_SERVERS}, and scale {KIE_SERVERS}.
+
+.Procedure
+. Enter your project in the OpenShift Web cluster console. 
+. In the OpenShift Web console navigation panel, select *Operators* and then *Subscriptions*.
+. Click the name of the subscription that containt `businessautomation`. Information about this subscription is displayed.
+. Under the *Installed version* heading, click the version name of the subscription. An overview of the subscription is displayed.
+. Select the *Instances* tab.
+. Click the name of a deployed environment. 
+. Select the *YAML* tab. A YAML source is displayed.
+. If you want to change common settings, such as passwords, edit the values under `commonConfig:`.
+. If you want to add new {KIE_SERVERS}, add their descriptions at the end of the block under `servers:`. For example:
```

Comment:
We supposedly should use colons with incomplete sentences, like "For example:", as we've discussed. Maybe "`servers:` as shown in the following example:" You use this frequently, and I actually don't disagree, just pointing it out so that you can choose to deliberately flout or address as needed.

Search all instances of "For/for example:" in the doc and address accordingly. Several instances.

Reply:
Changed this instance, because of awkward punctuation. but not really up to handling all of the others now as they are non-trivial to change. most are not operat-ros-ecific and a full review of the template docs is planned for week after next.